README.md

```diff
@@ -28,8 +28,9 @@ Our release cycle is independent of `react-native`. We follow semver and here is
 
 | `@react-native-community/cli`                                    | `react-native`   |
 | ---------------------------------------------------------------- | ---------------- |
-| ^4.0.0 (`next`)                                                  | ^0.63.0          |
-| ^3.0.0                                                           | ^0.61.0, ^0.62.0 |
+| ^5.0.0 (`next`)                                                  | master           |
```

Comment:
this should link to `next` branch

Reply:
Updated and added branches to few other releases too to make it more explicit.